src/index.js

```
@@ -126,16 +130,24 @@ class InfiniteCalendar extends Component {
+		// Make sure the initial selection (day or week) is positioned in view.
+		// The List's initial scrollTop covers the common case; this is a no-op
+		// when the selection is already visible.
+		let targetDate = this.state.selectedWeek || this.state.selectedDate;
+		if (targetDate && this.list) {
+			this.scrollToDate(targetDate, 0, false, false);
+		}
 	}
 
 	componentWillReceiveProps(next) {
 		let {min, minDate, max, maxDate, locale} = this.props;
 		let {display, isClickOnDatepicker, selectedWeek, selectedDate} = this.state;
```

ESLint

[error] 'isClickOnDatepicker' is defined but never used

(no-unused-vars)